### DISCUSSION TITLE
How to set default value for Composite types

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#15031] [Q&A] [answered]
Hi, I have a table like this, how can I set init data for languages field?
```
model User {
  id             String    @id @default(auto()) @map("_id") @db.ObjectId
  email          String    @unique
  languages      Translation []  @default([{ language:"en",slug:"abc", name:"English"  }])   // Can I set default data here?

type Translation {
  language Lang   @default(en)
  slug     String
  name     String @default("")
}

enum Lang {
  es
  en
}
```

**Comment by austincrim** [comment, marked as answer]
Hey @IRediTOTO 👋,

Prisma does not support this use case yet, would you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with more details?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
